job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContextSpec.groovy

```diff
@@ -1488,4 +1488,83 @@ class WrapperContextSpec extends Specification {
         'key1' | 'key2' | 'key3'  | null   | 'key1'      | 'key2'      | 'key3'      | ''
         'key1' | 'key2' | 'key3'  | 'key4' | 'key1'      | 'key2'      | 'key3'      | 'key4'
     }
+
+    def 'call versionNumber'() {
+        when:
+        context.versionNumber('', '') {
+        }
+
+        then:
+        with(context.wrapperNodes[0]) {
+            name() == 'org.jvnet.hudson.tools.versionnumber.VersionNumberBuilder'
+            children().size() == 11
+            children()[0].name() == 'versionNumberString'
```

Comment:
there is no need to reference all children by index, the order is not relevant in this case. See the other tests for the expected syntax, e.g. `versionNumberString[0].value() == ''`